File: src/main/java/de/sean/splugin/spigot/inventories/BlockLockInventory.java
package de.sean.splugin.spigot.inventories;

import de.sean.splugin.util.SUtil;
import org.bukkit.Bukkit;
import org.bukkit.Material;
import org.bukkit.inventory.Inventory;

public class BlockLockInventory {
    public static final String INVENTORY_NAME = "Block Lock";
    public static Inventory inventory = Bukkit.createInventory(null, 9, INVENTORY_NAME);
}

File: BlockProt/src/main/kotlin/de/sean/blockprot/bukkit/inventories/BlockLockInventory.kt
package de.sean.blockprot.bukkit.inventories

import de.sean.blockprot.util.Strings
import org.bukkit.Bukkit

object BlockLockInventory {
    val INVENTORY_NAME = Strings.getString("inventories.block_lock.name", "Block Lock")
    fun createInventory() = Bukkit.createInventory(null, 9 * 1, INVENTORY_NAME)
}
